classes/local/util/manager.php

```diff
@@ -139,6 +139,10 @@ protected function prepare_files_skeletons() {
         if ($this->should_have('settings')) {
             $this->prepare_file_skeleton('settings.php', 'php_internal_file', 'settings');
         }
+
+        if ($this->should_have('uninstall')) {
+            $this->prepare_file_skeleton('db/uninstall.php', 'php_internal_file', 'uninstall');
```

Comment:
By the way, I think the skel class should be called db_uninstall but it was too little detail to reject the pull request for.